launchers/qt/src/LauncherState.h

```diff
@@ -176,7 +176,7 @@ private slots:
     QString _displayName;
     QString _applicationErrorMessage;
     QString _currentClientVersion;
-    QString _buildTag { QString::null };
+    QString _buildTag { "" };
```

Comment:
```suggestion
    QString _buildTag { QString() };
```
`QString()` creates a null string, and a null default is needed for program logic: `Builds::getBuild(...)` tests `.isNull()` which is false if `_buildTag` is initialized to `""`.

Alternatively, remove the initializer which achieves the same thing.